unity-renderer/Assets/UnityGLTF/Scripts/GLTFComponent.cs

```diff
@@ -375,23 +367,16 @@ private void EnqueueDownload()
 
         private long animationsEstimatedSize;
         private long meshesEstimatedSize;
-        public long GetAnimationClipMemorySize()
-        {
-            return animationsEstimatedSize;
-        }
+        public long GetAnimationClipMemorySize() { return animationsEstimatedSize; }
 
-        public long GetMeshesMemorySize()
-        {
-            return meshesEstimatedSize;
-        }
+        public long GetMeshesMemorySize() { return meshesEstimatedSize; }
 
         private void OnDestroy()
         {
 #if UNITY_STANDALONE || UNITY_EDITOR
             if (DataStore.i.common.isApplicationQuitting.Get())
                 return;
 #endif
-            isDestroyed = true;
```

Comment:
What about the handling of the `isDestroyed` flag? We shouldn't be removing it altogether? As it is right now we are creating dead code with this change.